src/error.rs

```diff
@@ -112,6 +112,12 @@ pub enum SlateDBError {
 
     #[error("Database already exists: {msg}")]
     DatabaseAlreadyExists { msg: String },
+
+    #[error("Byte format version mismatch")]
```

Comment:
🤔 do we need to include expected_version and actual_version in this string in order to show up in the user facing message?

Reply:
Good catch. In the tests, I get:

```
Should succeed with valid flatbuffer data: InvalidVersion { expected_version: 1, actual_version: 2 }
```

But that's just dumping the error. I'll update the string.

Reply:
Fixed here:

845d912